### PR TITLE
introduce minimal lighttpd example that does not break static analysis

### PR DESCRIPTION
**Pull request by aneksteind**
As can be seen, not much works at the moment. But this gives us a starting point.

### CODE REVIEW

**Comment by kkysen** [comment]
I was going to put the different versions in separate crates.

**Comment by aneksteind** [comment]
@kkysen feel free to commit to this PR, my primary concern was putting up a minimal working example

**Comment by kkysen** [comment]
Okay, I'll move things around a bit and then merge it.

**Review by kkysen — APPROVED** [review]
Merging this now as is, then fixing things.

File: analysis/tests/lighttpd/src/lighttpd-minimal.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#![feature(rustc_private)]
```
Comment (kkysen) [inline analysis/tests/lighttpd/src/lighttpd-minimal.rs:1]: Why is this needed?  Can I delete it?
